cloki.js

```diff
@@ -109,8 +109,9 @@ const handler_404 = require('./lib/handlers/404.js').bind(this);
 fastify.setNotFoundHandler(handler_404);
 
 /* Hello cloki test API */
-const handler_hello = require('./lib/handlers/hello.js').bind(this);
+const handler_hello = require('./lib/handlers/ready').bind(this);
 fastify.get("/hello", handler_hello);
+fastify.get("/ready", handler_hello);
```

Comment:
Some clients use the /hello endpoint, we should decommission it at a later time. 
Could we keep /hello as an alias for the /ready handler?

Reply:
Exactly the thing I did.
Or you want this request to response "cloki"?